<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar Transacciones</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255);
color: black;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Deposito Bancario&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de Documento</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTipo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--- Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DNI - Documento nacional de identidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PSS - Pasaporte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VISA</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de Documento</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDocumento">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Monto en dolares</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMonto">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMotivo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--- Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Honorarios/Servicios tecnicos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Remesas/Giros</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donaciones</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Motivo del giro</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkTerminos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cliente acepta terminos y condiciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegistrar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 190, 111);</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/dd/banco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrimerNombre">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Primer Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSegundoNombre">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Segundo Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrimerApellido">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Primer Apellido</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSegundoApellido">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Segundo Apellido</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSexo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--- Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mujer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hombre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Otro</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Sexo</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="txtFecha">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/mm/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha de nacimiento</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLugar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--- Seleccione una opcion</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ciudad de nacimiento</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../b3ad3b2d/iconos.qrc"/>
  <include location="../d0c7d8b9/ds.qrc"/>
 </resources>
 <connections/>
</ui>
